includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1152,4 +1145,26 @@ private function create_token_from_setup_intent( $setup_intent_id, $user ) {
 			];
 		}
 	}
+
+	/**
+	 * Wrapper function to manage requests to WC_Stripe_API.
+	 *
+	 * @param string   $path   Stripe API endpoint path to query.
+	 * @param string   $params Parameters for request body.
+	 * @param WC_Order $order  WC Order for request.
+	 * @param string   $method HTTP method for request.
+	 *
+	 * @return object JSON response object.
+	 */
+	protected function stripe_request( $path, $params = null, $order = null, $method = 'POST' ) {
```

Comment:
Now we can mock out those static method calls from `WC_Stripe_API`. I tried a million different hacky ways to mock this class out and this was the cleanest...but if you disagree, don't hesitate to air your disapproval!

Reply:
_chefs kiss_